lib/context_adapter_helper.js

```diff
@@ -333,12 +408,13 @@ function getOperationDescriptor(request, callback) {
 /**
  * Checks that an queryContext response returned by the Context Adapter to get
  *  service information is well-formed
+ * @param {object} context The context for logging
  * @param {object} response The response received from the Context Adapter
  * @param {string} serviceId The identifier of the service whose information was retrieved
  * @param {function} callback Callback to be called with the result of the check
  * @return {*} True if the queryContextResponse is valid. False otherwise.
  */
-function checkQueryContextResponse(response, serviceId, callback) {
+function checkQueryContextResponse(context, response, serviceId, callback) {
```

Comment:
Why do you pass the logging context to the function? This is a bit confusing, and a bit dirty: I would limit the context to each module (to trace the logging of a request you can use domains), and I think the parameters of the function should reflect the information the function needs to do its work (and logging is a collateral task that shouldn't have a lot to do with the function).